cli/src/commands/transpile.ts

```diff
@@ -18,9 +18,9 @@
  *  For further information you can contact legal(at)bitloops.com.
  */
 
-import inquirer, { QuestionCollection } from 'inquirer';
+// import inquirer, { QuestionCollection } from 'inquirer';
```

Comment:
Do not leave commented out statements like this. Remove them completely. If we need them we can always go back in a previous version.